src/graphene-ray.c

```diff
@@ -52,6 +52,11 @@
 #include <math.h>
 #include <float.h>
 
+#if defined(_WIN32) && ! defined(isnanf)
```

Comment:
Shouldn't this be `defined(_MSC_VER)` since I think MinGW does provide `isnanf`?

Reply:
mingw-w64 does not provide isnanf

```
$ fgrep -r isnanf /mingw64/x86_64-w64-mingw32/include/
/mingw64/x86_64-w64-mingw32/include/intrin.h:    __MACHINECE(int _isnanf(float))
/mingw64/x86_64-w64-mingw32/include/math.h:  extern int __cdecl __isnanf (float);
/mingw64/x86_64-w64-mingw32/include/math.h:  __CRT_INLINE int __cdecl __isnanf (float _x)
/mingw64/x86_64-w64-mingw32/include/math.h:        __isnanf(x),                                            \
/mingw64/x86_64-w64-mingw32/include/math.h:   _CRTIMP int __cdecl _isnanf(float _X);
```

so the check must not be changed

Reply:
okay, thanks for checking :)